src/pages/EditReportFieldPage.tsx

```diff
@@ -86,6 +86,7 @@ function EditReportFieldPage({route, policy, report}: EditReportFieldPageProps)
 
     const handleReportFieldDelete = () => {
         ReportActions.deleteReportField(report.reportID, reportField);
+        setIsDeleteModalVisible(false);
         Navigation.dismissModal(report?.reportID);
```

Comment:
is this line actually doing anything then?

Reply:
Yes, IMO it dismiss the modal stack to avoid the cyclic dependency